app/controllers/projects/new_tree_controller.rb

```diff
@@ -6,15 +6,65 @@ def show
   end
 
   def update
-    file_path = File.join(@path, File.basename(params[:file_name]))
-    result = Files::CreateService.new(@project, current_user, params, @ref, file_path).execute
-
+	flag=0
```

Comment:
Use 2 (not -1) spaces for indentation.<br>Surrounding space missing for operator '='.<br>Tab detected.